src/lib/format/l20n/ast/parser.js

```diff
@@ -7,18 +7,20 @@ const MAX_PLACEABLES = 100;
 
 export default {
   parse: function(env, string, pos = false) {
```

Comment:
Can we make 'parse' the only exported symbol?

    export function parse(emit, string, pos = false) { ...}

This will allow us to do:

    import { parse } from 'format/l20n/ast/parser';